crates/src/lib.rs

```diff
@@ -156,32 +157,45 @@ impl DriftClient {
     /// * `markets` - list of markets to subscribe
     ///
     /// This is a no-op if already subscribed
-    pub async fn subscribe_markets(&self, markets: &[MarketId]) -> SdkResult<()> {
-        self.backend.subscribe_markets(markets).await
+    pub async fn subscribe_markets(
+        &self,
+        markets: &[MarketId],
+        on_account: Option<Arc<Box<OnAccountFn>>>,
+    ) -> SdkResult<()> {
+        self.backend.subscribe_markets(markets, on_account).await
```

Comment:
```suggestion
    pub async fn subscribe_markets_with_callback(
        &self,
        markets: &[MarketId],
        on_account: Option<Arc<Box<OnAccountFn>>>,
    ) -> SdkResult<()> {
        self.backend.subscribe_markets(markets, on_account).await
```

Reply:
would be good to keep the existing api unchanged, use new method only when user wants the callback